Source/r_plane.c

```diff
@@ -375,6 +377,9 @@ static void do_draw_plane(visplane_t *pl)
 
 	    // Vertical offset allows careful sky positioning.
 
+	    if (dc_texheight >= 200)
+	      dc_texturemid = s->rowoffset + 200*FRACUNIT;
```

Comment:
It seems that the author of Eviternity has set `rowoffset` everywhere, so the tall sky doesn't fill the entire screen. It's hard to say what the author's intention is, PrBoom+ just stretches in all cases: https://github.com/coelckers/prboom-plus/blob/5a5233a729fa65aa2d9e03a1cddffc7e4e0eb66c/prboom2/src/r_plane.c#L441-L445